cmd/utils.go

```diff
@@ -3,44 +3,58 @@ package cmd
 import (
 	"fmt"
 	"os"
-	"path/filepath"
-	"strings"
+
+	"github.com/bitrise-core/bitrise-init/scanners/ios"
+	"github.com/bitrise-core/bitrise-init/scanners/xamarin"
+	"github.com/bitrise-core/bitrise-init/utility"
 )
 
-// ProjectType ...
-type ProjectType string
+// IDEType ...
+type IDEType string
 
 const (
-	// iOS
-	xcodeWorkspace ProjectType = "*.xcworkspace"
-	xcodeProject   ProjectType = "*.xcodeproj"
+	xcodeIDE   IDEType = "iOS"
```

Comment:
`iOS` is not an IDE, i would revert this to projectType

Reply:
FIxed in:
https://github.com/bitrise-tools/codesigndoc/pull/95/commits/2026e7093d3e0ada66adb59c9605ad9ccc708441